sdk/core/azure-core/CHANGELOG.md

```diff
@@ -6,7 +6,9 @@
 
 - Cut hard dependency on requests library
 
-### Breaking Changes
+### Breaking Changes in the Provisional `azure.core.rest` package
+
+- Removed `chardet` dependency in `azure.core.rest`. Breaking because we removed the step that used `chardet` to inspect the response encoding
```

Comment:
I would phrase it as impact on customer, not internal details:

azure.core.rest won't try to guess the charset anymore if it was impossible to extract it from HttpResponse analysis

Reply:
Sounds good!